### DISCUSSION TITLE
請問有隧道配置ssh tunnel 配置流程文件嗎？

### DISCUSSION THREAD

**Discussion by dearsiang** [hhyo/Archery#2477] [General]
HI 大大

這邊最近想要用archery 隧道配置功能，Archery 安裝在A系統網路，要從A系統網路，去連另一個系統B的網路內的mongo db，這邊配置隧道配置的想法如下：

1.取得B系統網路的ssh server 的ssh key
2.配置在Archery 隧道配置
3.已確認從Archery 可以ssh 連得到Ｂ的ssh server

配置完後，無法連上，錯誤訊息如下
127.0.0.1:45451: timed out, Timeout: 30s, Topology Description: <TopologyDescription id: 659b79ed9b1a23b1b1867bc7, topology_type: Single, servers: [<ServerDescription ('127.0.0.1', 45451) server_type: Unknown, rtt: None, error=NetworkTimeout('127.0.0.1:45451: timed out')>]>

**Comment by Grain-Yu** [comment]
local port forwarding or  remote port forwarding? 参考这篇文章：https://zhuanlan.zhihu.com/p/469638489

**Comment by xiaoshumiao6** [comment]
ssh tunnel 可以正常支持mysql ， 貌似mongodb 是有问题的；我配置mongo 通过ssh tunnel 也是有问题的；  
https://github.com/hhyo/Archery/discussions/2546
